src/views/CoCurricularTranscript/guestTranscript.tsx

```diff
@@ -5,30 +5,31 @@ import { Participation } from 'services/membership';
 import { TranscriptItems } from 'services/transcript';
 
 const participation1: Participation = Participation.Member;
+const currentYear: string = new Date().getFullYear().toString();
```

Comment:
This means that participation2 will often end up being the coming fall instead of the past one. That doesn't really make sense if it is a future involvement. I would recommend setting everything back a year (subtract 1 from the year).

Reply:
Will do!